handsontable/src/renderers/numericRenderer/numericRenderer.js

```diff
@@ -53,6 +53,8 @@ export function numericRenderer(hotInstance, TD, row, col, prop, value, cellProp
     TD.dir = 'ltr';
   }
 
+  cellProperties.displayValue = newValue;
```

Comment:
I think that it's not the best idea to change cell meta within the renderer function. The changes disturb the function purity 🤔